core/platform/models.py

```diff
@@ -1,6 +1,6 @@
 # coding: utf-8
 #
-# Copyright 2014 The Oppia Authors. All Rights Reserved.
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Why do you modify this?

Reply:
Sorry SIr BImastakenly I deleted the agreement comment and when i copied it from other files of oppia I copied it since at the time my changes were not getting pushed.

Reply:
Now I have made that change.

Reply:
Not done.